packages/cardano-services/src/Asset/AssetHttpService.ts

```diff
@@ -0,0 +1,46 @@
+import * as OpenApiValidator from 'express-openapi-validator';
+import { AssetProvider, Provider } from '@cardano-sdk/core';
+import { HttpService } from '../Http';
+import { Logger } from 'ts-log';
+import { ServiceNames } from '../Program';
+import { providerHandler } from '../util';
+import express from 'express';
+import path from 'path';
+
+/**
+ * Dependencies that are need to create AssetHttpService
+ */
+export interface AssetHttpServiceDependencies {
+  /**
+   * The asset provider to fetch data
+   */
+  assetProvider: Provider & AssetProvider;
```

Comment:
```suggestion
  assetProvider: AssetProvider;
```
Let's merge https://github.com/input-output-hk/cardano-js-sdk/pull/346 first, then you can make this change